src/Components/__tests__/ArtsyShippingEstimate.jest.tsx

```diff
@@ -0,0 +1,154 @@
+import type { default as Arta } from "@artaio/arta-browser"
+import type ArtaEstimate from "@artaio/arta-browser/dist/estimate"
+import { screen, waitFor } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import { ArtsyShippingEstimate } from "Components/ArtsyShippingEstimate"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapperTL"
+import type { ArtsyShippingEstimate_Test_Query } from "__generated__/ArtsyShippingEstimate_Test_Query.graphql"
+import { useEffect, useState } from "react"
+
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+
+jest.mock("Utils/getENV", () => ({
+  getENV: jest.fn((() => "mock-api-key") as any),
+}))
+
+let mockArtaEstimate: ArtaEstimate
+
+beforeEach(() => {
+  mockArtaEstimate = {
+    open: jest.fn(),
+    close: jest.fn(),
+    isReady: true,
+    isOpen: false,
+    validate: jest.fn(() => Promise.resolve(undefined)),
+  } as unknown as ArtaEstimate
+})
+
+const mockArtaModule: typeof Arta = {
+  init: jest.fn(),
+  estimate: jest.fn(() => mockArtaEstimate),
+} as unknown as typeof Arta
+
+jest.mock("Utils/Hooks/useLoadScript", () => {
+  return {
+    useLoadScript: jest.fn(({ onReady }) => {
+      useEffect(() => {
+        ;(window as any).Arta = mockArtaModule
+        onReady()
+      }, [onReady])
+    }),
+  }
+})
+
+const TestWrapper = ({ artwork }) => {
+  const [show, setShow] = useState(true)
+
+  return (
+    <div>
+      {show ? (
+        <ArtsyShippingEstimate artwork={artwork} />
+      ) : (
+        <div>Unmounted</div>
+      )}
+      <div>
+        <button type="button" onClick={() => setShow(!show)}>
+          Toggle
+        </button>
+      </div>
+    </div>
+  )
+}
+
+const { renderWithRelay } =
+  setupTestWrapperTL<ArtsyShippingEstimate_Test_Query>({
+    Component: TestWrapper,
+    query: graphql`
+      query ArtsyShippingEstimate_Test_Query @relay_test_operation {
+        artwork(id: "example") {
+          ...ArtsyShippingEstimate_artwork
+        }
+      }
+    `,
+  })
+
+describe("ArtsyShippingEstimate", () => {
+  describe("with an artwork that cannot be estimated", () => {
+    const mockArtworkData = {
```

Comment:
#suggestion - maybe worth to be a bit more descriptive here. I would name it with some reference to why this can not be estimated. Like 'artworkMissingFullLocationData'? That is why it can not be estimated, right? 
Is that the case that Arta does not return estimate only based on country?